.changelog/26018.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+command: prevent panic on graceful shutdown
```

Comment:
This should be a user-facing changelog entry, not the developer-facing commit message. Something like:

```suggestion
agent: Fixed a bug where the agent could panic during graceful shutdown
```